File: JavaScript-Fundamentals/ConditionalStatements/02.MultiplicationSign/MultiplicationSign.js
// Write a script that shows the sign (+, - or 0) of the product of three real numbers, without calculating it.
// Use a sequence of if operators.

var jsConsole;
var btn = document.getElementById('check');
btn.onclick = getMultiplicationSign;

function getMultiplicationSign() {
	var firstNumber = jsConsole.readFloat('#firstNumber');
	var secondNumber = jsConsole.readFloat('#secondNumber');
	var thirdNumber = jsConsole.readFloat('#thirdNumber');
	var product = firstNumber * secondNumber * thirdNumber;

	if (product < 0) {
		jsConsole.writeLine("Multiplication sign is -");
		console.log("Multiplication sign is -");
	} else if (product > 0) {
		jsConsole.writeLine("Multiplication sign is +");
		console.log("Multiplication sign is +");
	} else {
		jsConsole.writeLine("Multiplication sign is 0");
		console.log("Multiplication sign is 0");
	}	
}

File: CSharp part1/05. Conditional-Statements/04.MultiplicationSign/MultiplicationSign.cs
﻿/* Write a program that shows the sign (+, - or 0) of the product of three real numbers, without calculating it.
Use a sequence of if operators. */

using System;

class MultiplicationSign
{
    static void Main()
    {
        Console.Write("Enter a:");
        double a = double.Parse(Console.ReadLine());
        Console.Write("Enter b:");
        double b = double.Parse(Console.ReadLine());
        Console.Write("Enter c:");
        double c = double.Parse(Console.ReadLine());
        double product = a * b * c;
        if (product < 0)
        {
            Console.WriteLine("-");
        }
        else if (product > 0)
        {
            Console.WriteLine("+");
        }
        else 
        {
            Console.WriteLine("0");
        }
    }        
}

